roles/strimzi_kafka/defaults/main.yaml

```diff
@@ -0,0 +1,11 @@
+kafka_cluster_name: seldon
```

Comment:
Maybe also adjust prefix to all variables to be `strimzi_kafka_` instead of just `kafka_`? 
Not a must though - don't have strong opinion on that.

Reply:
Done